external/nmea/parse.c

```diff
@@ -283,19 +283,28 @@ int nmea_parse_GPGST( const char *buff, int buff_sz, nmeaGPGST *pack )
 
   nmea_trace_buff( buff, buff_sz );
 
+  char type;
+
   if ( 8 != nmea_scanf( buff, buff_sz,
```

Comment:
you need to bump these expected result numbers too... have you tested this?

Reply:
not at all. I don't have any means to test this, so the idea was to ship and let a user with the possibilities test the nightlies. Sorry for not being more helpful.

Reply:
... I hope to be able to work on unit tests for this as part of something that could hit our pipeline.

Reply:
:smile: (just don't let @roya0045 hear that... shhhhhh :shushing_face: )

Reply:
@nyalldawson I'm not sure if I shoul be insulted by this, but its fine. We can all be bumblefucks at times.

Reply:
I think it was a friendly pat to mostly remind me of shortcomings I've imputed to others 😜

Reply:
Don't be... I'm insulting @m-kuhn and calling him out for hypocrisy, not insulting you! 😁 (And @m-kuhn knows me well enough to know that 90% of what I say is firmly tongue in cheek!!)

Reply:
This is what I was thinking. No offence was taken ;)  I am also well aware of my shortcoming and always find refreshing to see someone else do things blindly from time to time.